libraries/botbuilder-dialogs-adaptive/src/adaptiveDialog.ts

```diff
@@ -344,237 +325,89 @@ export class AdaptiveDialog<O extends object = {}> extends Dialog<O> {
         }
     }
 
-    private async queueFirstMatch(planning: PlanningContext, event: DialogEvent): Promise<boolean> {
-        const memory = this.getMemoryForEvent(planning, event);
+    private async queueFirstMatch(sequence: SequenceContext, event: DialogEvent, preBubble: boolean): Promise<boolean> {
         for (let i = 0; i < this.rules.length; i++) {
-            const changes = await this.rules[i].evaluate(planning, event, memory);
+            const changes = await this.rules[i].evaluate(sequence, event, preBubble);
             if (changes && changes.length > 0) {
-                changes[0].turnState = memory.turn;
-                planning.queueChanges(changes[0]);
+                sequence.queueChanges(changes[0]);
                 return true;
             }
         }
 
         return false;
     }
 
-    private async queueBestMatches(planning: PlanningContext, event: DialogEvent): Promise<boolean> {
```

Comment:
Love deleting this code :) it was annoying to debug

Reply:
me too :)